services/base-jdbc/src/main/java/org/eclipse/hono/service/base/jdbc/config/JdbcProperties.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2020, 2023 Contributors to the Eclipse Foundation
+ * Copyright (c) 2020, 2024 Contributors to the Eclipse Foundation
```

Comment:
Would you mind removing the second year (in this case: 2024) in the copyright header of all the files you changed in this PR? The year in which the file was originally created is sufficient now (see e.g. https://github.com/eclipse-hono/hono/pull/3632)

Reply:
Yes most definitely, I removed all those current years now.